def/configmap/main.go

```diff
@@ -38,7 +38,7 @@ type options struct {
 	output    string
 }
 
-// multiKeyValue allows --key=value --key=value
+// multiKeyValue allows --key=value
```

Comment:
I think changing this to `--key=value --key=value2` would better capture the intent of the comment, I believe